cpp/main.cc

```diff
@@ -420,6 +400,25 @@ int main(int argc, char **argv) {
         }
         TString sample_name = sample_names[sample];
         int sample_procid = sample_procids[sample];
+        
+        // Skip signal samples still in production
+        if ( year == "2017" ) {
+          if ( sample == "NMSSM_XToYHTo2G2B_MX-360_MY-100" ||
```

Comment:
I'm actually always a bit confused about how to use the sample names for signal. If I only runs one signal locally, I used the command '' ./main.exe test 2018 0 0 1 NMSSM_XToYHTo2G2B_MX-360_MY-100 '' It will tell me no samples selected. I'm wondering what is the correct way to specify a signal sample name?

Reply:
ok, thank you for clarification!